src/Dependency/ExportedNode/ExportedAttributeArgumentNode.php

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Dependency\ExportedNode;
+
+use JsonSerializable;
+use PHPStan\Dependency\ExportedNode;
+
+class ExportedAttributeArgumentNode implements ExportedNode, JsonSerializable
+{
+
+	public function __construct(
+		private ?string $name,
+		private string $value,
+		private bool $byRef,
```

Comment:
TODO: Most likely not even possible, test it

Reply:
I don't think we need the name here, being indexed correctly by `int|string` in ExportedAttributeNode should be fine.

Reply:
I don't think so, you can switch named argument and keep the value & index.

`#[Column(nullable: true)]`
`#[Column(unique: true)]`

Reply:
I removed the src/Dependency/ExportedNode/ExportedAttributeArgumentNode.php entirely as it was storing only the expression value. Let me know if you'd prefer to keep it